config/core.js

```diff
@@ -49,7 +52,7 @@ module.exports = {
       WETH_ADDRESS,
     ],
     stableTokenAddresses: [USDT_ADDRESS, USDC_ADDRESS],
-    nativePricePool: '0x72d8e6d7307dcf11a3456b9f6fdfad05385b2f3e', // WCORE/USDT
```

Comment:
You want to save this in a comment too? so we know how we achieve pricing for another time? WDYT